handlers/meta/handlers.go

```diff
@@ -1090,6 +1091,10 @@ func (h *handler) requestWAC(payload whatsapp.SendRequest, accessToken string, r
 		return courier.ErrResponseUnparseable
 	}
 
+	if slices.Index([]int{4, 80007, 130429, 131048, 131056, 133016}, respPayload.Error.Code) > 0 {
```

Comment:
Matching the error codes https://developers.facebook.com/docs/whatsapp/cloud-api/support/error-codes#throttling-errors to all be marked as throttle error and we retry up to 3 times

Reply:
this is odd way to check if x in y. Save the list of error codes as a global, then use `slices.Contains`